modules/qsort/test/test_qsort.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 Taktaev Artem
+#include <gtest/gtest.h>
+
+#include <algorithm>
+#include <vector>
+
+#include "include/qsort.h"
+
+TEST(QSort_Test, Test_Creating_Vector_Exception) {
+    int n = -1;
+    ASSERT_ANY_THROW(std::vector<double> vec = QSort::createRandomVector(n));
```

Comment:
Присваивание лишнее. Да и n, в принципе, тоже.
```suggestion
    ASSERT_ANY_THROW(QSort::createRandomVector(-1));
```